contribs/gmf/less/mobile.less

```diff
@@ -253,3 +253,20 @@ button[ngeo-mobile-geolocation] {
     }
   }
 }
+
+//mobile styles for the theme and background selector
+.gmf-theme-selector,
+.gmf-mobile-background-layer-selector {
+  .gmf-icon-check:after {
+    font-size: 1.2rem;
+    position: relative;
+    bottom: 0.8rem;
+  }
+  .gmf-text {
+    margin: 3% 10px 0 0;
+  }
+  .gmf-thumb {
+    height: auto;
+    width: 25%;
```

Comment:
Effectively I just see it on the phone/tablet size, and it's wheal, previously I test it on desktop and the icons will already be on the new line ...